example/src/shared/participantsWindowManager.ts

```diff
@@ -0,0 +1,212 @@
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { getNumberOfCurrentlyVisiblePlaces } from '@utils';
+import { useRef } from 'react';
+
+type ParticipantWithTrack = {
+  participant: Membrane.Participant;
+  trackId?: string;
+};
+
+type LRUNode = {
+  lastActive: number;
+  isActive: boolean;
+  index: number;
+  // Local user and screenshare tracks
+  // are not movable and should stay on top.
+  isMovable: boolean;
+};
+
+// Used to manage order in which participants tracks are displayed.
+export const ParticipantsTracksWindowManager = (
+  maxNumUsersOnScreen: number
+) => {
+  // This string array will containg strings in a form of participantId+trackId
+  // to differentiate between camera and screenshare video tracks.
+  const participantsOrder = useRef<string[] | null>(null);
+  const lru = useRef<LRUNode[]>([]);
+
+  // First not movable tracks(local and screenshares) then least recently used.
+  const lruComparator = (a: LRUNode, b: LRUNode) => {
+    if (a.isMovable) {
+      if (b.isMovable) {
+        return a.lastActive > b.lastActive ? 1 : -1;
+      } else {
+        return -1;
+      }
+    } else if (b.isMovable) {
+      return 1;
+    }
+    return 0;
+  };
+
+  // This pushes screenshares to the top.
+  const orderByScreenshare = (
+    a: ParticipantWithTrack,
+    b: ParticipantWithTrack
+  ) => {
+    if (a.participant.type === 'Local') {
+      return -1;
+    } else if (b.participant.type === 'Local') {
+      return 1;
+    }
+
+    if (
+      a.participant.tracks.find((t) => t.id === a.trackId)?.metadata.type ===
```

Comment:
you can add a function that for given participant checks if it's a screenshare?

Reply:
Done.